_static/js/custom.js

```diff
@@ -291,6 +291,19 @@ $(document).ready(() => {
     }
   }
 
+  // Change indentation from spaces to tabs for codeblocks.
+  const codeBlocks = document.querySelectorAll('.rst-content div[class^="highlight"] pre');
+  for (const codeBlock of codeBlocks) {
+    const classList = codeBlock.parentNode.parentNode.classList;
+    if (classList.contains('highlight-csharp') || classList.contains('highlight-python')) {
+      // CSharp and Python use spaces for indentation, ignore.
```

Comment:
I wonder if we should check for GDScript instead. This seems like a possible footgun when adding/changing languages, and I'm not sure spaces in random codeblocks should always be converted (shaders? Stuff that isn't code but formatted as code due to reasons?)

Besides that concern, this looks really good to me. :) Thanks for improving upon it still while we didn't even get to merging the first one.

Reply:
The main two I had in mind was GDScript and, to a lesser extent, C++. It should be simple enough to change the checks to opt-in for those exclusively, instead of the current opt-out format

Reply:
Updated to opt-in format!

Also, changed code-block formatting in a few areas from "C++" to "cpp" so that the filter would properly recognize them

Reply:
That was quick! Thanks, looks good 🙏